lib/HTTP/Tiny.pm

```diff
@@ -1792,10 +1805,15 @@ certificate has been verified by a CA. Assuming you trust the CA, this will
 protect against L<machine-in-the-middle
 attacks|http://en.wikipedia.org/wiki/Machine-in-the-middle_attack>.
 
-Certificate verification requires a file containing trusted CA certificates.
+Certificate verification requires a file or directory containing trusted CA
+certificates.
+
+If C<IO::Socket::SSL::default_ca()> is available, HTTP::Tiny will use it to
+detect the default location of your CA certificates. This function also supports
```

Comment:
Hmm... I think it was the "function" here that threw me off. Maybe s/function// should be enough to make it clearer (for me!)

Reply:
Fixed